src/components/ResultsList.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+
+class ResultsList extends Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			results: [],
+		};
+		
+		this.displayList = this.displayList.bind(this);
+		this.tableSort = this.tableSort.bind(this);
+	}
+
+	componentWillReceiveProps(nextProps) {
+		this.setState({ results: nextProps.results });
+	}
+
+	displayList() {
+		if (this.state.results.length === 0) {
+			return (
+				<tr>
+					<td colspan="5"><p>There are no results to display.</p></td>
+				</tr>
+			);
+		}
+
+		return this.state.results.map(result => {
+			return (
+				<tr>
+					<td>{ result.FirstName }</td>
+					<td>{ result.LastName }</td>
+					<td>{ result.PhoneNumber }</td>
+					<td>{ result.Email }</td>					
+					<td>{ result.FormName }</td>
+				</tr>
+			);
+		});
+	}
+
+	tableSort(fieldName) {
+		const results = Array.prototype.slice.call(this.state.results);
+
+		results.sort((left, right) => {
```

Comment:
maybe move the actual sort inner function to it's own function. I think we have been trying to do that?

Reply:
I was just being lazy. I'll do that.